### PR TITLE
Adding Alwasy Online feature to Caching API

### PR DESCRIPTION
**Pull request by shivaleelac**
## PR summary

**Fixes:** <! -- link to issue -->

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] New tests
- [ ] Build/CI related changes
- [x] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by KumarGanesanIBM** [comment]
@nimisha0092 
looks like, there was an intermittent error in Page Rule IT. It passed now.
can you please add the ssl certificate skip statement (which we did in internal repo) as part of this PR..?
Thanks,

**Review by KumarGanesanIBM — APPROVED** [review]
Looks good to me.

File: cachingapiv1/caching_api_v1.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2020.
+ * (C) Copyright IBM Corp. 2021.
```
Comment (kennburger) [inline cachingapiv1/caching_api_v1.go:2]: If this was originally released in 2020, It should read '2020, 2021'.

File: cachingapiv1/caching_api_v1_suite_test.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2020.
+ * (C) Copyright IBM Corp. 2021.
```
Comment (kennburger) [inline cachingapiv1/caching_api_v1_suite_test.go:2]: Same comment here ...  2020, 2021

File: cachingapiv1/caching_api_v1_test.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2020.
+ * (C) Copyright IBM Corp. 2021.
```
Comment (kennburger) [inline cachingapiv1/caching_api_v1_test.go:2]: 2020, 2021

**Review by kennburger — CHANGES_REQUESTED** [review]

**Review by kennburger — APPROVED** [review]
LGTM

**Review by ajay-malhotra1 — APPROVED** [review]
lgtm

**Pull request merged**